tests/MongoDB.Driver.Tests/AggregateFluentTests.cs

```diff
@@ -196,6 +196,55 @@ public void ChangeStream_should_add_the_expected_stage_when_options_is_null()
             result.Should().Be(1);
         }
 
+        [SkippableFact]
+        public void Group_with_accumulator_should_return_expected_result()
+        {
+            RequireServer.Check().Supports(Feature.AggregateAccumulator);
+
+            var client = CreateClient();
+            var databaseName = "db";
+            var collectionName = "collection";
+            DropCollection(client, databaseName, collectionName);
+            var collection = client.GetDatabase(databaseName).GetCollection<BsonDocument>(collectionName);
+            var books = new[]
+            {
+                BsonDocument.Parse("{ _id : 8751, title : 'The Banquet', author : 'Dante', copies : 2 }"),
+                BsonDocument.Parse("{ _id : 8752, title : 'Divine Comedy', author : 'Dante', copies : 1 }"),
+                BsonDocument.Parse("{ _id : 8645, title : 'Eclogues', author : 'Dante', copies : 2 }"),
+                BsonDocument.Parse("{ _id : 7000, title : 'The Odyssey', author : 'Homer', copies : 10 }"),
+                BsonDocument.Parse("{ _id : 7020, title : 'Iliad', author : 'Homer', copies : 10 }")
+            };
+            collection.InsertMany(books);
+
+            var initFunction = "function() { return { count : 0, sum : 0 } }";
+            var accumulateFunction = "function(state, numCopies) { return { count : state.count + 1, sum : state.sum + numCopies } }";
+            var mergeFunction = "function(state1, state2) { return { count : state1.count + state2.count, sum : state1.sum + state2.sum } }";
+            var finalizeFunction = "function(state) { return (state.sum / state.count) }";
+            var accumulatorBody = new BsonDocument
+            {
+                { "init", initFunction },
+                { "accumulate", accumulateFunction },
+                { "accumulateArgs", new BsonArray { "$copies" } },
+                { "merge", mergeFunction },
+                { "finalize", finalizeFunction },
+                { "lang", "js" }
+            };
+            var accumulatorExpression = new BsonDocument("$accumulator", accumulatorBody);
+            var groupProjection = new BsonDocument
```

Comment:
Random thought: Should we include this example in the refdocs? 🤔 💭

Reply:
as I understand we didn't do it before so I don't see why it should be done now. But I don't mind. Probably it's a question for @jyemin ?

Reply:
but to be honest, since we even don't have any implementation for this feature I don't see what exactly we should add into docs.. NOTE: we have a number (a lot?) of similar cases without any mentions in the docs

Reply:
It's true we often add features without updating our refdocs. In this case, I think I'd be happy with just a link that points to this unit test as an example :) We've done similar things before in the past I think. Perhaps a new Jira ticket to add a paragraph to the refdocs and to link to this example? In any case, I defer to you, my LGTM still stands! 😸